ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -122,18 +139,37 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
     ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
     ///   - title: The title string.
     ///   - accessoryView: An optional accessory view that appears near the trailing edge of the view.
+    ///   - leadingView: An optional custom view that appears near the leading edge of the view.
     /// - Returns: a value representing the calculated preferred width of the view.
-    @objc public class func preferredWidth(style: Style, title: String, accessoryView: UIView? = nil) -> CGFloat {
-        let titleSize = title.preferredSize(for: Constants.titleTextStyle.font)
+    @objc public class func preferredWidth(style: Style, title: String, accessoryView: UIView? = nil, leadingView: UIView? = nil) -> CGFloat {
+        var font = Constants.titleTextStyle.font
+        if style == .headerPrimary {
+            font = Constants.primaryTitleTextStyle.font
+        }
+        let titleSize = title.preferredSize(for: font)
 
         var width = Constants.horizontalMargin + titleSize.width + Constants.horizontalMargin
 
         if let accessoryView = accessoryView {
             width += Constants.accessoryViewMarginLeft + accessoryView.frame.width
         }
-
+        
+        if let leadingView = leadingView {
+            width += Constants.accessoryViewMarginLeft + leadingView.frame.width
+        }
+                
         return width
     }
+    
+    private static func titleLeftOffset(leadingView: UIView? = nil) -> CGFloat { //sopleTODO
```

Comment:
no it should be "leading" instead of left. have you tested RTL? also, what's the TODO for?